src/xpk/core/pathways.py

```diff
@@ -60,6 +60,8 @@ def get_pathways_proxy_args(args) -> str:
   yaml = """- --server_port=29000
               - --resource_manager_address={rm_address}
               - --gcs_scratch_location={args.pathways_gcs_location}"""
+  if args.additional_pw_proxy_args:
+    yaml += args.additional_pw_proxy_args.replace(' ', '\n' + ' '*14 +'- ')
 
```

Comment:
Maybe:
```suggestion
additional_pw_proxy_string = ''
if args.additional_pw_proxy_args:
  additional_pw_proxy_string = #construct your string here
 yaml = """
                   ...
                   - {intendation_string}{additional_pw_proxy_string}
 
```

Reply:
Indentation is what defines structure in a yaml, so the flag I'm appending needs to follow whichever indentation the current line has. What I was looking for is some way to avoid the magic 14, which might become a different number if the peripheral indentation changes for whatever reason. I'll follow your suggested pattern for clarity though.